src/index.test.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react'
+import { create, act } from 'react-test-renderer'
```

Comment:
I would much more prefer using the React Testing Library to avoid accessing `props` and stuff of React elements in tests.

Reply:
I have no idea about testing stuff in React, I just followed [this guide on Jest](https://jestjs.io/docs/tutorial-react) basically. `react-test-renderer` also [seems to be part of the official testing utils provided by React itself](https://reactjs.org/docs/test-renderer.html). Are you referring to [this testing library](https://github.com/testing-library/react-testing-library)?

Reply:
>  Are you referring to this testing library?

Yes.

Reply:
ok I will re-write tests using that library instead. on that note, is it a better idea to actually use [react-hooks-testing-library](https://github.com/testing-library/react-hooks-testing-library) instead of react-testing-library?

Reply:
ok so I switched the testing library to [@testing-library/react-hooks](https://github.com/testing-library/react-hooks-testing-library) and the end result is much cleaner.